src/App.js

```diff
@@ -0,0 +1,11 @@
+import './App.css';
+import Calculator from './components/calculator';
+
+function App() {
+  const newCalculator = new Calculator();
+  return (
+    newCalculator.render()
+  );
+}
```

Comment:
- [ ] According to requirements, app.js should be class-bassed too, please change it. 🐱

Reply:
Prefect, thanks for remember me that, now is fixed 👍